Add quickstart guide for TS client (#256)
dydxprotocol/v4-documentation: pages/api_integration-guides/Getting_started_with_the_typescript_client.mdx

import { Tab, Tabs } from "nextra-theme-docs";

# Getting Started with the TypeScript Client

## Quickstart

This is a quickstart guide to get you up and running with the TypeScript client library for dYdX V4.
For more guides and code examples, please refer to the [Typescript client repository](https://github.com/dydxprotocol/v4-clients/tree/main/v4-client-js). 

### Installation

The `v4-client-js` uses node `v18` for development. 
You can use nvm to manage different versions of node.

```bash
nvm install
nvm use
nvm alias default $(nvm version) # optional

# You can run the following commands to ensure that you are running the correct `node` and `npm` versions.
node -v # expected: v18.x.x (should match .nvmrc)
npm -v  # expected: 8.x.x
```

> Now install the V4 Client

Reviewer: i see so these instructions refer to installing `V4 Client` rather than implementing on the V4 client. These are two different things (e.g. "integrating JS client" and "implementing on the JS client". I think the former is more important probably.

In the former case (integrating), the above instructions of setting up `nvm` etc. are not needed. They should set up their own package/workspace, where they install the V4 client using the below command `npm i @dydxprotocol/v4-client-js`. I believe the node version doesn't matter, and the package should work for all workspaces with node >=18

npm version doesn't matter in that case

import { Tab, Tabs } from "nextra-theme-docs";

# Getting Started with the TypeScript Client

## Quickstart guides

> There are multiple quickstart guides to get you up and running with the TypeScript client library for dYdX V4. 

Reviewer: ```suggestion
There are multiple quickstart guides to get you up and running with the TypeScript client library for dYdX Chain. 
```

### Run examples from the v4-client-js repository
- Clone and run the examples from the v4-client-js repository. See the following [guide](https://github.com/dydxprotocol/v4-clients/tree/main/v4-client-js#readme) to run TS client examples from the [Typescript client repository](https://github.com/dydxprotocol/v4-clients/tree/main/v4-client-js). 

### Setup your own project

> - Install and use the v4-client-js code as a package (in your own software) for your own [example project](https://github.com/dydxfoundation/solutions-public/tree/main/v4-client-js/examples), which you can modify freely
> and iterate on for your API trading use cases.

Reviewer 2: Nitpick, not blocking: suggest rephrasing to separate the directive (install and use...) and the example project. As written, "for your own example project" isn't quite right (although intuitive enough that I doubt it will cause confusion)

```suggestion
- Install and use the v4-client-js code as a package in your own software. You can modify this [example project](https://github.com/dydxfoundation/solutions-public/tree/main/v4-client-js/examples) to work for your API trading use cases.
```
